router/regtest.go

```diff
@@ -38,14 +46,18 @@ func (r *RegTest) New() error {
 	if err != nil {
 		return err
 	}
-	addr, err := client.GetAccountAddress("wallet.dat")
+
+	// Setup db
+	db := &Database{}
+	err = db.New()
 	if err != nil {
 		return err
 	}
 
 	r.Client = client
-	r.FaucetAccount = addr.String()
+	r.DB = db
 
+	// Since regtest chain should be empty at this point, lets mine some blocks to increase miner balance
 	mine(r, 200)
```

Comment:
better doing it outside the New function

Reply:
consider that this should be done every time the dockerized daemon cluster is stared. I put it here so this is automated instead of doing it manually or calling a function after the regtest client is initialized with `New()`

Reply:
That's fine. I mean we neeld to call mine outside at the same level of New (after). So it can be only a constructor function without side effects